doc/developer-documentation/forgae-history/history.md

```diff
@@ -0,0 +1,17 @@
+# forgae-history
+
+## History of your deploys
+
+In order to see a list of what you've deployed you can run the following command:
```

Comment:
is this part of the forgae-logger? if yes, please provide info and requirements about it